### ISSUE TITLE
changed connection array and doesn't kept backward compatibility

### ISSUE THREAD

**Issue by jitchavan** [senecajs/seneca-mongo-store#37]
Merge pool request https://github.com/senecajs/seneca-mongo-store/commit/94658224dd2d1fb88aa7f65ac67f257f425c795a
just stopped our connection and throwing errors directly.

Connection Syntax Changes such as this should be added only after **BOLD** notifications and on **Readme** also.
Also plugin should give proper **errors** to show that  Connection Syntaxes changed.

we just spend a lot of time to dig on this. hope this should not happen to others.

**Comment by mcdonnelldean** [comment]
@jitchavan Appologies to hear this. My understanding is this change is not on NPM as of yet. Are you working against `master`? If so I would suggest against this as back compat is only considered in versions, not on the live branch.

Either way. I want to ensure the version that does go out is backwards compatible and will try get master updated soon. Thanks for raising this.

**Comment by mihaidma** [comment]
Merged a fix that could be one of the issues you encountered: https://github.com/senecajs/seneca-mongo-store/pull/46

**Comment by mihaidma** [comment]
Made another PR with a test that connects both with params: https://github.com/senecajs/seneca-mongo-store/pull/47
There are also tests that connect with URI with and without `native_parser: true`
I'll close this, please reopen if issues persist
